pkg/ecdsa/tss/protocol_announce.go

```diff
@@ -81,6 +100,18 @@ func AnnounceProtocol(
 
 	switch ctx.Err() {
 	case context.DeadlineExceeded:
+		for _, member := range keepMemberAddresses {
+			_, announcedPresence := receivedMemberIDs[member]
```

Comment:
What about the current member? Will it be included in `keepMemberAddresses` and `receivedMemberIDs`?

Reply:
The current member also writes to this channel, so yes, it will be included there.